js/models/Profile.js

```diff
@@ -84,17 +54,23 @@ export default class extends BaseModel {
     colorFields.forEach((colorField) => {
       const clr = attrs[colorField];
 
-      if (typeof clr !== 'undefined' && is.not.hexColor(clr)) {
-        addError(colorField, 'Please provide a valid hex color.');
+      if (is.not.hexColor(clr)) {
+        addError(colorField, app.polyglot.t('profileModelErrors.provideValidHexColor'));
+      } else if (clr.charAt(0) !== '#') {
+        addError(colorField, 'The color should start with a leading hash.');
       }
     });
 
     if (attrs.email && is.not.email(attrs.email)) {
-      addError('email', 'Please provide a valid email.');
+      addError('email', app.polyglot.t('profileModelErrors.provideValidEmail'));
     }
 
     if (attrs.website && is.not.url(attrs.website)) {
-      addError('website', 'Please provide a valid url.');
+      addError('website', app.polyglot.t('profileModelErrors.provideValidURL'));
+    }
+
+    if (attrs.handle && attrs.handle.charAt(0) === '@') {
+      addError('handle', 'The handle should not start with a leading hash.');
```

Comment:
ok, but it is, IMO adding unneeded bloat to the translation file. An example is there are 2 validations on the social account type: a.) Please provide a type b.) Type must be one of the required types.

Since the UI represents the field as a drop down and it populates the drop down from the values stored in the model, there's no practical way the user will ever see these errors. The only way there would be would be is if, for example, we allowed the user to update this value from some other form and the developer didn't represent it as a drop down, in which case I think it's a dev error that would need to be fixed.